src/io/dsv.rs

```diff
@@ -309,35 +309,35 @@ Boston;United States;4628910
         );
     }
 
-    /// Ignored as normalisation is not done yet completely
-    #[ignore]
     #[test]
     fn csv_with_various_different_constant_and_literal_representations() {
         let data = "\
-a;b;c
-Boston;United States;4628910
-<Dresden>;Germany;1234567
-My Home Town;Some<where >Nice;2
-Trailing Spaces do not belong to the name   ; What about spaces in the beginning though;123
-\"\"\"Do String literals work?\"\"\";\"\"\"Even with datatype annotation?\"\"\"^^<http://www.w3.org/2001/XMLSchema#string>;456
-The next column is empty;;789
+a;b;c;d
+Boston;United States;Some String;4628910
+<Dresden>;Germany;Another String;1234567
+My Home Town;Some<where >Nice;<https://string.parsing.should/not/change#that>;2
+Trailing Spaces do not belong to the name   ; What about spaces in the beginning though;  what happens to spaces in string parsing?  ;123
+\"\"\"Do String literals work?\"\"\";\"\"\"Even with datatype annotation?\"\"\"^^<http://www.w3.org/2001/XMLSchema#string>;\"\"\"even string literals should just be piped through\"\"\"^^<http://www.w3.org/2001/XMLSchema#string>;456
+The next 2 columns are empty;;;789
 ";
 
         let expected_result = [
-            ("Boston", "United States", 4628910),
-            ("Dresden", "Germany", 1234567),
-            ("My Home Town", "\"Some<where >Nice\"", 2),
+            ("Boston", "United States", "Some String", 4628910),
+            ("Dresden", "Germany", "Another String", 1234567),
+            ("My Home Town", "\"Some<where >Nice\"", "<https://string.parsing.should/not/change#that>", 2),
```

Comment:
I think using a raw string literal might make this more readable:

```rust
("My Home Town", r#""Some<where >Nice""#, "<https://string.parsing.should/not/change#that>", 2),
```

Reply:
Good idea, thanks!